### PR TITLE
White paper and some more details for the E2EE

### PR DESCRIPTION
**Pull request by jospoortvliet**
added technical details & illustrations and the whitepaper for download.

@nextcloud/website

### CODE REVIEW

**Comment by jospoortvliet** [comment]
it is on staging:
https://nextcloud.com/endtoend/?staging=true

File: page-endtoend.php [hunk 1]
```diff
@@ -174,8 +164,96 @@
...
+        </div>
+        <div class="col-md-10 col-md-offset-1">
+            <p class="section--paragraph__tittle revealOnScroll"><?php echo $l->t('Encrypting files');?></p>
+            <p class="section--paragraph revealOnScroll">
+                <?php echo $l->t('Creating an end-to-end encrypted folder is done by flagging an empty folder as end-to-end encrypted, creating metadata, encrypting and uploading it. The process is protected by locking, with the client requesting a lock from the server and releasing it again after all changes are finished and uploaded.');?>
+            </p>
+            <p class="section--paragraph revealOnScroll">
+                <?php echo $l->t('Files are each encrypted with their own, unique key, which is stored in the metadata file. All data in the metadata file is, in turn, securely encrypted and files names and directory structures are also hidden from the server using a unique UUID as name for each file.');?>
+            </p>
+            <p class="section--paragraph__tittle revealOnScroll"><?php echo $l->t('Download our white paper for more details!');?></p>
```
Comment (MariusBluem) [inline page-endtoend.php]: Make it a call to action...

Comment (jospoortvliet) [reply page-endtoend.php]: well the whitepaper is a bit higher up, I could link to but that is a pain.

**Comment by blizzz** [comment]
image not loaded

spectacle p27092

File: page-endtoend.php (hunk 1 above)
Comment (MariusBluem) [reply page-endtoend.php]: Then leave the sentence at all 🙃

Comment (jospoortvliet) [reply page-endtoend.php]: I just removed it, it is obvious and mentioned before already.

**Comment by jospoortvliet** [comment]
@blizzz good catch, added the image
@MariusBluem removed that sentence, it was the 3rd time we mentioned it - enough is enough...

shift-refresh on the page should show the changes. No other issues?

**Comment by splitt3r** [comment]
Don´t know if it´s already on your todo but the site title should be changed.

**Pull request merged**